src/builders/questionnaireBuilder.ts

```diff
@@ -0,0 +1,39 @@
+import { R4 } from '@ahryman40k/ts-fhir-types';
+
+export class QuestionnaireBuilder {
+  libraryName: string;
+  questionnaire: R4.IQuestionnaire;
+
+  constructor(libraryName: string) {
+    this.libraryName = libraryName;
+    this.questionnaire = {
+      resourceType: 'Questionnaire',
+      id: libraryName,
+      title: libraryName,
+      status: R4.QuestionnaireStatusKind._draft,
+      extension: [
+        {
+          url: 'http://hl7.org/fhir/StructureDefinition/cqf-library',
+          valueCanonical: `Library/${libraryName}`
+        }
+      ],
+      item: []
+    };
+  }
+
+  addCqfExpression(expression: string): void {
+    this.questionnaire.item!.push({
+      linkId: expression,
+      extension: [
+        {
+          url: 'http://hl7.org/fhir/StructureDefinition/cqf-expression',
+          valueExpression: {
+            language: R4.ExpressionLanguageKind._textCql,
+            expression: `"${this.libraryName}".${expression}`
+          }
+        }
+      ],
+      type: R4.Questionnaire_ItemTypeKind._string
```

Comment:
We probably need to figure what to do about type.  We probably want to use choice with the with the expectation that the cql expression defines the answerOption filed to choose a value(s) from that would be references to the FHIR resources returned from the expression